src/css/dashboard/update_notifier.scss

```diff
@@ -14,7 +14,7 @@
   align-items: center;
```

Comment:
Please could you move this file one directory up and include it from the relevant scss file there because it's no longer part of the dashboard.